capstone/backend/src/main/java/com/google/sps/spanner/UsersSpanner.java

```diff
@@ -0,0 +1,210 @@
+    System.err.println("    SpannerExample updateusingpartitioneddml my-instance step39-db");
+    System.err.println("    SpannerExample deleteusingpartitioneddml my-instance step39-db");
+    System.err.println("    SpannerExample updateusingbatchdml my-instance step39-db");
+    System.err.println("    SpannerExample createtablewithdatatypes my-instance step39-db");
+    System.err.println("    SpannerExample writedatatypesdata my-instance step39-db");
+    System.err.println("    SpannerExample querywitharray my-instance step39-db");
+    System.err.println("    SpannerExample querywithbool my-instance step39-db");
+    System.err.println("    SpannerExample querywithbytes my-instance step39-db");
+    System.err.println("    SpannerExample querywithdate my-instance step39-db");
+    System.err.println("    SpannerExample querywithfloat my-instance step39-db");
+    System.err.println("    SpannerExample querywithint my-instance step39-db");
+    System.err.println("    SpannerExample querywithstring my-instance step39-db");
+    System.err.println("    SpannerExample querywithtimestampparameter my-instance step39-db");
+    System.err.println("    SpannerExample clientwithqueryoptions my-instance step39-db");
+    System.err.println("    SpannerExample querywithqueryoptions my-instance step39-db");
+    System.err.println("    SpannerExample createbackup my-instance step39-db");
+    System.err.println("    SpannerExample listbackups my-instance step39-db");
+    System.err.println("    SpannerExample listbackupoperations my-instance step39-db");
+    System.err.println("    SpannerExample listdatabaseoperations my-instance step39-db");
+    System.err.println("    SpannerExample restorebackup my-instance step39-db");
+    System.exit(1);
+  }
+  
+  public static void main(String[] args) throws Exception {
+    if (args.length != 3) {
+      printUsageAndExit();
+    }
+    // [START init_client]
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    try {
+      String command = args[0];
+      DatabaseId db = DatabaseId.of(options.getProjectId(), args[1], args[2]);
+      // [END init_client]
+      // This will return the default project id based on the environment.
+      String clientProject = spanner.getOptions().getProjectId();
+      if (!db.getInstanceId().getProject().equals(clientProject)) {
+        System.err.println(
+            "Invalid project specified. Project in the database id should match the"
+                + "project name set in the environment variable GOOGLE_CLOUD_PROJECT. Expected: "
+                + clientProject);
+        printUsageAndExit();
+      }
+      // Generate a backup id for the sample database.
+      String backupName =
+          String.format(
+              "%s_%02d",
+              db.getDatabase(), LocalDate.now().get(ChronoField.ALIGNED_WEEK_OF_YEAR));
+      BackupId backup = BackupId.of(db.getInstanceId(), backupName);
```

Comment:
You don't need this backup? it's not being used anywhere. Can also remove from the `run()`